### DISCUSSION TITLE
Energy efficient CPU for many cameras

### DISCUSSION THREAD

**Discussion by miczlo** [blakeblackshear/frigate#7438] [Ask A Question]
I've been reading the HW guides. I have 12 cameras, 10 are 5MP and 2 are 4K. It's ok now on a 7th gen i7 and a coral but I'm considering moving frigate to some other hardware.
In terms of CPU is quick sync low power CPU like n100 ok or do I need something more powerful in terms of frequency and core count?
I'm also wondering if I could get away with one hardware for both Plex and frigate

**Comment by NickM-27** [comment]
I really don't think an n100 would be enough for plex and frigate. From what I have seen, AMD ryzen is much more power efficient than intel. Something like the 5600G with built in graphics would be worth looking into as well.

**Reply by miczlo** [reply]
So is Frigate more raw CPU power intensive or does it rely more on the igpu?

**Reply by NickM-27** [reply]
You definitely want an igpu for handling decoding video and something for running object detection. Other than that the main resource user is just motion detection on the CPU and that's not too demanding but it depends on the detect resolution.
